lib/rules/length-zero-no-unit/__tests__/index.js

```diff
@@ -318,7 +326,7 @@ testRule({
 		},
 		{
 			code: 'a { top: 0.000px; }',
-			fixed: 'a { top: 0; }',
+			fixed: 'a { top: 0.000; }',
```

Comment:
Should have been outside the scope of this rule in favour of [number-no-trailing-zeros](https://stylelint.io/user-guide/rules/number-no-trailing-zeros).

Reply:
Agreed. 👍🏼 

So, this change may be a bug fix, not a refactoring?
(maybe, this PR title is confusing... 🤔 )